### PR TITLE
Dispatch static event so that RenderCache and OpenLayers can be statical...

### PR DESCRIPTION
**Pull request by fabianx-ai**
...ly inited.

### CODE REVIEW

File: service_container.module [hunk 1]
```diff
@@ -48,7 +48,7 @@ function service_container_ctools_plugin_type() {
  * Implements hook_ctools_plugin_directory().
  */
 function service_container_ctools_plugin_directory($owner, $plugin_type) {
-  if ($owner == 'service_provider') {
+  if ($owner == 'service_container') {
```
Comment (fabianx-ai) [inline service_container.module:51]: This is an unrelated bug fix.

Comment (dawehner) [inline service_container.module:51]: ooh, I fixed that on d.o already, damnit! :(
